packages/bundle-size/src/utils/collectLocalReport.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs').promises;
+const glob = require('glob');
+const path = require('path');
+const { findGitRoot, findPackageRoot } = require('workspace-tools');
+
+/** @typedef {import('../utils/buildFixture').BuildResult} BuildResult */
+
+/** @typedef {BuildResult & { packageName: string }} BundleSizeReportEntry */
+/** @typedef {BundleSizeReportEntry[]} BundleSizeReport */
+
+/**
+ * @param {string} reportFile
+ *
+ * @return {Promise<[string, BuildResult[]]>}
+ */
+async function readReportForPackage(reportFile) {
+  const reportFilePath = path.resolve(process.cwd(), reportFile);
+
+  const packageName = path.basename(/** @type {string}*/ (findPackageRoot(reportFilePath)));
+  const packageReport = JSON.parse((await fs.readFile(reportFilePath)).toString());
```

Comment:
as this returns `any` its a good common practice to explicitly annotate the expected return call

```js
/** @type {BuildResult[]} */
const packageReport = JSON.parse(await fs.readFile(reportFilePath, 'utf8'));
```

Reply:
Applied in 2841356.